### PR TITLE
fix: my updates subscription regression

### PR DESCRIPTION
**Pull request by dolcalmi**

### CODE REVIEW

File: src/graphql/root/subscription/my-updates.ts [hunk 1]
```diff
@@ -108,7 +108,10 @@ const MeSubscription = {
       })
     }
 
-    return pubsub.asyncIterator([USER_PRICE_UPDATE_EVENT, walletUpdateEvent(ctx.uid)])
+    return pubsub.asyncIterator([
+      USER_PRICE_UPDATE_EVENT,
+      walletUpdateEvent(ctx.wallet.user.walletId),
```
Comment (nicolasburtey) [inline src/graphql/root/subscription/my-updates.ts:113]: not sure what ctx here is, but couldn't it be just ctx.walletId? (assuming ctx is an InvoiceUser object?)

Comment (vindard) [reply src/graphql/root/subscription/my-updates.ts:113]: If I understand correctly `ctx` is [this object](https://github.com/GaloyMoney/galoy/blob/398d149f02f53d4d70c79244b57ff9d0331974f3/src/servers/graphql-server.ts#L115) with the following properties:
```
{
        logger,
        uid: userId,
        wallet,
        // FIXME: we should not return this for the admin graphql endpoint
        domainUser,
        domainAccount,
        user,
        geetest,
        account,
        ip,
}
```

In that object `wallet` still comes from `WalletFactory` and is a `LightningUserWallet` and I also don't see any other place we could access the `walletId` from as yet.

Comment (nicolasburtey) [reply src/graphql/root/subscription/my-updates.ts:113]: domainAccount.defaultWalletId?

Comment (vindard) [reply src/graphql/root/subscription/my-updates.ts:113]: Or that 🙈

Comment (dolcalmi) [reply src/graphql/root/subscription/my-updates.ts]: updated and tested

**Review by vindard — APPROVED** [review]

**Review by samerbuna — APPROVED** [review]

**Pull request merged**